west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8fb465f43f6a90ac4c4ce89265047e04f7d5ee5b
+      revision: pull/1021/head
```

Comment:
This is no longer needed, right? Since we pulled in the kconfig here.

Reply:
ping @sigvartmh

Reply:
https://github.com/nrfconnect/sdk-zephyr/pull/1021#discussion_r1062614160

I'm not sure what the decision point should be. I sort of agree keeping it in the NCS repository will be easier with regards to upmerges. However he has a point when it comes to being easier to localize the change.